src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Default {
+    private double size;
+    private double side1;
+    private double side2;
+    private double length;
+
+    public IsoscelesTrapezoid(String color, double size, double side1,
+                              double side2, double length) {
+        super(color);
+        this.size = size;
+        this.side1 = side1;
+        this.side2 = side2;
+        this.length = length;
+    }
+
+    @Override
+    public double getArea() {
+        return size * size;
+    }
+
+    @Override
+    public double getProperty() {
+        return 0.5 * (side1 * side2) * length;
```

Comment:
The method name getProperty() is not descriptive of its purpose. Consider renaming it to something more meaningful, like getPerimeter(), if it is intended to calculate the perimeter.